CONTRIBUTING.md

```diff
@@ -1,36 +1,142 @@
-# Contributing
+# Contributing Guide
 
-First read the overall project contributing guidelines. These are all
-included in the qiskit documentation:
+To contribute to Qiskit Experiments, first read the overall project contributing guidelines.
+These are included in the qiskit documentation:
 
 https://qiskit.org/documentation/contributing_to_qiskit.html
 
 ## Contributing to Qiskit Experiments
 
-In addition to the general guidelines there are specific details for
-contributing to qiskit experiments, these are documented below.
+In addition to the general guidelines, the specific guidelines for
+contributing to Qiskit Experiments are documented below.
+
+### Contents
+
+  + [Choose an issue to work on](#choose-an-issue-to-work-on)
+  + [Pull request checklist](#pull-request-checklist)
+  + [Code style](#code-style)
+  + [Test your code](#test-your-code)
+    - [STDOUT/STDERR and logging capture](#stdout-stderr-and-logging-capture)
+  + [Changelog generation](#changelog-generation)
+  + [Release notes](#release-notes)
+    - [Adding a new release note](#adding-a-new-release-note)
+      * [Linking to issues](#linking-to-issues)
+    - [Generating release notes](#generating-release-notes)
+  + [Documentation](#documentation)
+  + [Adding deprecation warnings](#adding-deprecation-warnings)
+  + [Development cycle](#development-cycle)
+  + [Branches](#branches)
+  + [Release cycle](#release-cycle)
+  + [Proposing a new experiment](#proposing-a-new-experiment)
```

Comment:
Maybe proposing a new experiment should be a bit higher in the hierarchy? I don't know how common it is for your contributors to want to do this, but it seems perhaps more logical to move this section up maybe just after the `Choose an issue to work on` section?

Reply:
Good point, I actually think putting it first makes the most sense in terms of workflow.